--- oxidation_tweek_bg.js
+++ oxidation_tweek_bg.js
@@ -1,8 +1,9 @@
 let imports = {};
 imports['./oxidation_tweek'] = require('./oxidation_tweek');
 
-            const bytes = require('fs').readFileSync('oxidation_tweek_bg.wasm');
+            const join = require('path').join;
+            const bytes = require('fs').readFileSync(join(__dirname, 'oxidation_tweek_bg.wasm'));
             const wasmModule = new WebAssembly.Module(bytes);
             const wasmInstance = new WebAssembly.Instance(wasmModule, imports);
             module.exports = wasmInstance.exports;
-        
\ No newline at end of file
+        
